web/v1/snowflake/sql-runner/configs/datamodeling.json

```diff
@@ -0,0 +1,58 @@
+{
+  "schema": "iglu:com.snowplowanalytics.datamodeling/config/jsonschema/1-0-0",
+  "data": {
+    "dagName": "standard-webmodel",
+    "enabled": true,
+    "storage": "Default",
+    "sqlRunner": "0.9.3",
+    "lockType": "soft",
```

Comment:
Agreed. I think the thinking in that decision for the other DBs is that until they configure email alerts, it should be hard locked. When they configure email alerts they (or we) can swap it to soft locks.

Open to challenge on that plan.

Reply:
I totally agree on the consistency argument. My reasoning here is that since the model is safe to rerun, maybe it is worth denoting that too, or at least add this to the documentation.

Reply:
I think it's worth changing this to `hard` - simply because if it's left on `soft` and deployed, that can generate a lot of annoying notifications.